UI.tsv

```diff
@@ -325,7 +325,7 @@ UI_20150317_000324	{@st56}Dead{/}
 UI_20150317_000325	Spell Power
 UI_20150317_000326	Defense
 UI_20150317_000327	Magic Resistance
-UI_20150317_000328	Critical Rate
+UI_20150317_000328	Critical Attack Chance
```

Comment:
![critrate](https://cloud.githubusercontent.com/assets/10456776/10979135/9627a8fc-83af-11e5-9a86-abe40092bd6c.png)

As shown in the screenshot above, the value of Critical Rate or Critical Attack Chance can actually go higher than 100. Now what are people's opinions? For me, chance is a probability term and is associated with percentage values. @stvkno If you check the games you listed that use Chance. They specifically use Chance with percentages, which gets capped at 100%. WoW and Elder Scrolls both have something called Critical Strike Rating, which simply is a value like TOS that gets converted into a percentage.
An example of a game that uses Chance but as a percentage: http://leagueoflegends.wikia.com/wiki/Critical_strike#Chance
A link which someone clearly defines Crit Rate and Crit Chance for Tera Online (not sure if the same applies to ToS): https://www.reddit.com/r/TeraOnline/comments/2tl03o/whats_the_difference_between_crit_rate_and_crit/

What I feel is that ToS is using not using the Critical Rate value directly to determine whether you landed a critical hit, but rather the value gets inputted into a formula which then spits out a critical hit chance. We can always consider **Critical Hit Rate**?

The current ingame Help section seems not that useful atm. So I do welcome IMC to spruce it up xD.